lib/Dancer2/Manual.pod

```diff
@@ -2551,6 +2535,19 @@ list context) or a hash reference (in scalar context) to all defined
 parameters. Check C<param> below to access quickly to a single parameter
```

Comment:
This is part of the top level documentation for `param`/`params`, we wanted to keep this I thought for user's who have older Dancer2 apps.